riot/lol/model.go

```diff
@@ -118,164 +120,204 @@ type MiniSeries struct {
 
 // Match contains information about a match
 type Match struct {
+	Metadata *MatchMetadata `json:"metadata"`
```

Comment:
All of these should get more documentation once they release documentation for them officially